AUTHORS.txt

```diff
@@ -23,7 +23,9 @@ Moritz Jung
 Richard Schlitz
 Manuel Zahn
 Mikhaël Myara
+Domenic Prete
 Mathieu Jeannin
+Domenic Prete
```

Comment:
Domenic Prete repeated twice.
Could you please, update also docs/about/authors.rst?